03_010/client/src/utils/myUtils.ts

```diff
@@ -0,0 +1,18 @@
+const _ = {
+  $: <T>(
+    selector: string,
+    $target: HTMLElement | Document
+  ): (T extends HTMLElement ? T : any) | null => {
+    if ($target === document) return document.querySelector(selector);
+    else if ($target) return $target.querySelector(selector);
+    else throw new Error('해당 엘리먼트를 찾을 수 없습니다.');
```

Comment:
```suggestion
  $: <T>(
    selector: string,
    $target: HTMLElement | Document = document
  ): (T extends HTMLElement ? T : any) | null => {
    return $target.querySelector(selector);
```
이렇게 표현해주면 어떨까요?
$target의 기본 값으로 document를 넣어주면 될 것 같아요.
그리고 $target을 못 찾을 때 throw를 사용하는 것은 무척 위험한 행위가 아닐까요?
런타임에서 불필요한 에러가 발생할 수도 있을 것 같아요!